### ISSUE TITLE
Division by zero when there are no relevant expressions

### ISSUE THREAD

**Issue by jsmaniac** [florence/cover#118]
I have a file consisting mostly of macro definitions, and this triggers a division by zero in `html.rkt`:

https://github.com/florence/cover/blob/master/cover/private/html/html.rkt#L154

```
/: division by zero
  context...:
   /home/travis/.racket/snapshot/pkgs/cover/cover/private/html/html.rkt:145:0: make-html-file
   /home/travis/.racket/snapshot/pkgs/cover/cover/private/html/html.rkt:52:4: for-loop
   /home/travis/.racket/snapshot/pkgs/cover/cover/private/html/html.rkt:46:0: get-files
   /home/travis/.racket/snapshot/pkgs/cover/cover/private/html/html.rkt:30:0: generate-html-coverage
   /home/travis/.racket/snapshot/pkgs/cover/cover/raco.rkt:83:4
   /home/travis/.racket/snapshot/pkgs/cover/cover/private/shared.rkt:28:0: with-intercepted-logging
   (submod /home/travis/.racket/snapshot/pkgs/cover/cover/raco.rkt main): [running body]
   /home/travis/racket/collects/raco/raco.rkt: [running body]
```

For example, see this failed build on travis-ci: https://travis-ci.org/jsmaniac/phc/builds/118010454#L2993

As can be seen on coveralls.io, the file `graph/graph-test.rkt` has 0 relevant lines (sort by relevant lines to see it in the lsit): https://coveralls.io/jobs/12998765

**Comment by florence** [comment]
This was closed by #119? right

**Comment by jsmaniac** [comment]
@florence Yes, I'm closing this. Thanks!
